### PR TITLE
DISTX-717 Fixed authz issues for CustomConfigurations operations, add…

### PR DESCRIPTION
**Pull request by aakashshankar**
…ed corresponding real UMS tests.

Fixes [DISTX-717](https://jira.cloudera.com/browse/DISTX-717)

### Overview of changes
 - Entitlement added to real UMS tests tenant via [DISTX-722](https://jira.cloudera.com/browse/DISTX-722)
 - Modifications to remove dependency issues.

### Testing
- Added real UMS test to verify working.

See detailed description in the commit message.

### CODE REVIEW

**Review by aszegedi — APPROVED** [review]
By testing point of view LGTM. Please wait for review by @horadla23 as well before merge.

**Review by horadla23 — APPROVED** [review]
LGTM

**Pull request merged**
